src/main/scala/org/broadinstitute/dsde/firecloud/model/ModelJsonProtocol.scala

```diff
@@ -138,14 +138,14 @@ object ModelJsonProtocol {
   implicit object impQueryMap extends JsonFormat[QueryMap] {
     override def write(inputmap: QueryMap): JsValue = inputmap match {
       case matchall : ESMatchAll => matchall.toJson
-      case wildcard : ESWildcard => wildcard.toJson
+      case wildcard : ESMatch => wildcard.toJson
       case _ => throw new SerializationException("unexpected QueryMap type")
     }
 
     override def read(json: JsValue): QueryMap = {
       json.asJsObject.fields.keys.head match {
         case "match_all" => impESMatchAll.read(json)
-        case "wildcard" => impESWildcard.read(json)
+        case "wildcard" => impESMatch.read(json)
```

Comment:
I think this should be `case "match"` if I'm reading the `QueryMap` trait right.